src/lib/write-package.ts

```diff
@@ -122,7 +122,7 @@ export const writePackage = async (
       ),
       statusCode: 500,
     };
-    res.end(ret.error);
+    res.end(JSON.stringify(ret));
```

Comment:
Here and below - can we use `res.json` instead?

Reply:
I'm ok with using res.json. 

I like to imagine that using methods only available on http servers makes things easier to move around but in practice no one ever does.

Reply:
Heh, I come from the other side where I learned express far before the base level HTTP server, so I never know what's express and what's node, like the new jquery vs javascript